packages/main/src/Slider.hbs

```diff
@@ -0,0 +1,38 @@
+<div 
+	id="{{this._id}}"
+	?disabled= "{{disabled}}"
+	value = "{{value}}"
+	label = "{{label}}"
+	?expanded= "{{_expanded}}"
+	@click = {{_onclick}}
+	@keydown={{_onkeydown}}
+	class = "ui5-slider"
+	data-sap-focus-ref
+	@focusin="{{_onfocusin}}"
+	@focusout="{{_onfocusout}}"
+	tabindex="0"
+>
+	<div class="ui5-slider-header-block">
+		<div id="{{this._id}}--label" class="ui5-slider-label"> {{label}}</div>
+		<div class="ui5-slider-invisible-text"></div>
+		<ui5-button class="ui5-slider-arrow" icon="arrow-top" @click={{_onArrowUp}} tabindex="-1"></ui5-button>
```

Comment:
you depend on ui5-button, you have to import the Button.js and await for ui5-button definition, otherwise if someone uses the slider and happens not to use the Button this will fail

Reply:
Done